test/example/c-api-kitchen-sink.c

```diff
@@ -1603,8 +1603,9 @@ void test_core() {
 
 void test_unreachable() {
   BinaryenModuleRef module = BinaryenModuleCreate();
+  BinaryenAddTable(module, "tab", 0, 100, BinaryenTypeFuncref());
```

Comment:
Is this meant to be a 64-bit table?

Reply:
No, that shouldn’t be valid. I wouldn’t be surprised if we’re missing validation for it, though.

Reply:
Hmm.. well in order to make this more of a no-op change I updated the validator such that this test doesn't need changing.

We can make this change as a followup perhaps.

Reply:
This change is no longer part of this PR.